README.md

```diff
@@ -18,3 +18,13 @@ Setup
 -----
 
 The infrared sensor should be mounted in such a way that all balls entering the ball pit pass in front of it. The `BALL_DISTANCE_THRESHOLD` variable should be configured appropriately for the position of the sensor.
+
+Modes
+-----
+
+* Normal mode:
+    Counts the amount of balls successfully get
+* Timed mode:
+    Counts how many balls you can get within a time limit (60s)
+* Target mode:
+    Times how long it takes you to successfully get a target number of balls (50)
```

Comment:
Use Markdown syntax for this; something like `* **Normal mode:** Counts the amount of total amount of balls sensed.`.